### ISSUE TITLE
Documentation: Explain what 'Digg style' and 'Twitter style' mean

### ISSUE THREAD

**Issue by andybak** [shtalinberg/django-el-pagination#22]
I visited Digg and can't work out how it differs from Twitter's ajax endless scrolling.

Also - you say in the docs: "If Ajax pagination is not needed"

What does it even mean to have endless pagination without Ajax? 

Ideally a demo would be great but failing that - screenshots for Digg/Twitter with and without Ajax would be really illuminating.

**Comment by tisdall** [comment]
Yeah, I'm also lost as to the difference...  This project is forked from an older one and I'm wondering if Digg has changed their pagination since.  Doing a Google search it seems that Digg used to have a numbered pagination bar similar to what Google has on standard search results.

**Comment by shtalinberg** [comment]
I will try to set up a test project (maybe this week) on a sub-domain of the organization. Then you can watch how the current version.

**Comment by tisdall** [comment]
Screenshots should be sufficient, but then you have to update those when changes occur.  The major thing is, I don't think Digg actually has "Digg style pagination" as defined by this project (any more).

**Comment by shtalinberg** [comment]
How would it be called right?

**Comment by tisdall** [comment]
You could call it "old style Digg".  The major thing is not the naming but the lack of anything other than a name to explain what it is.  That's why I suggested a screenshot, so people would be able to figure out the difference from the picture.

**Comment by shtalinberg** [comment]
differences pictures are not enough to be understanding  (work via ajax)
best way for seen it's goto contributing  http://django-el-pagination.readthedocs.io/en/latest/contributing.html  and run the Django development server via $ make server  for comparison

**Comment by tisdall** [comment]
I think the point was that pictures would be better than saying "Digg style" when Digg no longer uses that style...  At worst, saying "Digg style" is now misleading.

**Comment by digicase** [comment]
How about removing references to Digg or Twitter, and use 'numbered' or 'infinite scroll' instead to describe the pagination styles.

**Comment by jedie** [comment]
Yes, it's IMHO more confusion.

It would be also great to have screenshots or better: Have a demo page.

I saw the example in the unittests. What's about to made it easier to run this examples locally?

**Comment by thelinuxkid** [comment]
@shtalinberg I was able to get the server running using the instructions on https://django-el-pagination.readthedocs.io/en/latest/contributing.html. However, I had to install Django manually with `pip install django` before running `make server`.

@andybak @tisdall, etc, here are screenshots of the two styles:
[Twitter Style](https://ibb.co/icEz5L)
[Digg Style](https://ibb.co/kRDsQL)

**Comment by andybak** [comment]
@thelinuxkid Thanks! Never expected to have this answered after 2 and half years. :)
